packages/radio-box-group/src/RadioBox.tsx

```diff
@@ -233,9 +237,14 @@ export default function RadioBox({
     showFocus,
   });
 
+  const tooltipTriggerHandlers = getTooltipTriggerHandlers<HTMLLabelElement>(
+    rest as HTMLProps<HTMLLabelElement>,
+  );
+
   return (
     <label
       {...radioBoxWrapper.prop}
+      {...tooltipTriggerHandlers}
```

Comment:
The cause of the bug was that we are passing `rest` into the `<input>` element, but the tooltip trigger handlers need to be passed into the component root

Reply:
Nice - this sounds like it might've been a hard one to catch lol.

I'm not sure if I'm a fan of the design of this code here, since we're passing a `rest` to a function only for it to explicitly define exactly what parameters it's expecting. I think it would be a cleaner and simpler solution for us to just explicitly pass these into the `label`, or spread them after defining them in a separate object like `const tooltipProps = { onClick, onMouseEnter, ... }; ... <label {...tooltipProps }>`